srcpkgs/ddccontrol/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ddccontrol'
+pkgname=ddccontrol
+version=0.4.4
+revision=1
+build_style=gnu-configure
+configure_args=" --disable-gnome --disable-gnome-applet --disable-dependency-tracking"
+hostmakedepends="automake tar libtool"
```

Comment:
`gettext-devel` is needed here as well? instead? as/of `makedepends`

Reply:
yes. https://travis-ci.org/github/void-linux/void-packages/jobs/714610166#L835

Reply:
@Gottox I got confused, autopoint issues were something else. Just need to add it to hostmakedepends.